models/volunteering-request-offer-base.js

```diff
@@ -1,9 +1,7 @@
-const mongoose = require('mongoose');
-
-const Schema = mongoose.Schema;
+const { Schema, model } = require('mongoose');
```

Comment:
If you change the import type, why don't you use it like:
```
import { Schema, model } from 'mongoose';
```
?

Reply:
you cannot use import export in pure js (for now available only as experimental option since node 10, more about that there: https://medium.com/@giltayar/native-es-modules-in-nodejs-status-and-future-directions-part-i-ee5ea3001f71 ) and i don't want to configure babel only for for this....
+ all this project working on commonjs (require)